CMakeLists.txt

```diff
@@ -256,3 +256,16 @@ set_target_properties(at_boost_program_options
    PROPERTIES COMPILE_FLAGS "-Wno-overloaded-virtual"
 )
 
+##
+## URLS
+##
+set(BOOST_URL_ENCODE_SOURCES
+  libs/url/src/encoding_opts.cpp
```

Comment:
boost/libs/url/src has more `cpp`s. Is just adding this one enough?

Reply:
It is for using the `boost::urls::encode` function, which is all I need for Yuba.

I figured in the future if we ever need the rest of the Boost URLs library functionality we could just add them then.
But I'm open to opinions, doesn't really matter all that much

Reply:
I'd vote for adding all now. Is build time a concern?

Reply:
Nope not a concern, I'll add them all then, and update the name to be `at_boost_url`

Reply:
Updated @artasano